build/pages/about/USC-main.html

```diff
@@ -0,0 +1,160 @@
+        </header>
+        <div class="container container--header_accord">
+            <div class="header_accord" data-js="header_accord">
+                <menu class="link_list link_list--info" data-js="info_menu">
+                    <h1 class="link_list__hdng">Information:</h1>
+                    <li><a href="/announcements">Announcements</a></li>
+                    <li><a href="/events">Events</a></li>
+                    <li><a href="/organizations">Organizations</a></li>
+                    <li><a href="/usc">About the USC</a></li>
+                    <li><a href="/usc/officers">Officers of the USC</a></li>
+                </menu>
+                <menu class="link_list link_list--srvcs" data-js="srvcs_menu">
+                    <h1 class="link_list__hdng">Services:</h1>
+                    <li><a href="/services/volunteer">Volunteer Opportunities</a></li>
+                    <li><a href="/services/education">Education and Training</a></li>
+                    <li><a href="/usc/contact">Contact the USC</a></li>
+                    <li><a href="/usc-cite/contact">Contact the USC-CITE</a></li>
+                </menu>
+                <form class="search_form" action="/search" method="post" data-js="search_form">
+                    <label for="search" class="search_form__label">Search the USC Infosite</label>
+                    <input id="search" class="search_form__input" type="text" />
+                    <button class="search_form__btn" type="submit">
+                        <img alt="" src="/build/assets/icons/magnifier.svg" /><span class="sr_only">Search the USC Infosite</span>
+                    </button>
+                </form>
+            </div>
+        </div>
+
+        <main class="main">
+            <h1 class="main__heading">The University Student Council</h1>
+            <div class="main__container main__container--about-USC">
+                <img src="/build/assets/logos/USC-logo.png" alt="" class="USC-logo">
+                <div class="main__container--text-content">
+                    <h2 class="main__sub-heading">What is the University Student Council (USC)?</h2>
+                    <p class="main__text">The University Student Council is the supreme student organization of the West Visayas State University herein referred to as the USC. The USC shall have the jurisdiction over the entire student body of the West Visayas State University. The functions of the student councils should be such that through them students shall be able to exercise academic freedom and to apply academic means, participate in maintaining and improving the environment in which the freedom of each is not tempered by equality of all, and in the process develop their leadership potentials and values needed in nation-building.</p>
+                </div>
+            </div>
+
+            <div class="main__container main__container--USC-officers">
+                <h2 class="main__sub-heading">USC OFFICERS</h2>
```

Comment:
I'll change this text too to reflect the little changes I'll make on the officers page.